### PR TITLE
Adding subtle cape flow animation.

### PR DESCRIPTION
**Pull request by peytonchance**
This PR adds an animation to blow Florence's cape lightly in the wind.

florence-anim-1

### CODE REVIEW

File: src/images/florence.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...4.5 39.6-4.4 1.8-1.7-7.3-7.5-8.7-9.6 0-.1-15 8.3-28.9 8.3z"/><path class="st0" d="M139.2 199.9h-7.1c-1.9 0-3.4-1.5-3.4-3.4V172c0-1.9 1.5-3.4 3.4-3.4h7.1c1.9 0 3.4 1.5 3.4 3.4v24.6c0 1.8-1.5 3.3-3.4 3.3z"/><path class="st0" d="M151.3 180.7v7.1c0 1.9-1.5 3.4-3.4 3.4h-24.6c-1.9 0-3.4-1.5-3.4-3.4v-7.1c0-1.9 1.5-3.4 3.4-3.4H148c1.8 0 3.3 1.5 3.3 3.4z"/><path class="st10" d="M134.5 54c9 0 17.8 1.1 26.1 3.1 2.4.6 5-.3 6.4-2.2l7.7-10.5c1.5-2.1 1.2-4.9-.8-6.6-10.2-8.8-24.1-14.2-39.4-14.2-15.5 0-29.5 5.5-39.7 14.4-2 1.7-2.3 4.5-.8 6.6l7.6 10.4c1.4 1.9 4 2.7 6.4 2.2 8.5-2.1 17.3-3.2 26.5-3.2z"/><path class="st14" d="M135.8 54c9 0 17.8 1.1 26.1 3.1 2.4.6 5-.3 6.4-2.2l4.4-5.9c-11-5.1-24.2-6.8-37.6-4.1C125 47 116 51.3 108.6 57.2c.3 0 .5-.1.8-.1 8.4-2 17.3-3.1 26.4-3.1z"/><path class="st0" d="M134.8 30.8c15.1 0 29.4 5.1 39.8 13.5l.8-1c1.6-2.1 1.2-5-.8-6.8-10.5-9-24-14.5-39.7-14.5-15.8 0-30.8 5.6-41.3 14.8-2.1 1.8-2.4 4.6-.8 6.8l1.8 2.5c.1-.1.2-.3.4-.4 10.4-9.3 24-14.9 39.8-14.9z"/><path d="M127.7 124.3c4.5 1.6 9.5 1.5 13.5-.7l.2-.1c-1.6-.8-3.8-1.2-6.2-1.2-3.1 0-5.8.8-7.5 2z" fill="#e75f67"/><path class="st14" d="M197.8 279.3c0 1.9-1.3 3.4-3 3.4h-52.7c26.3-10.7 42.8-17.2 54.3-34l1.4 30.6z"/></svg>
```
Comment (jeremyfrank) [inline src/images/florence.svg:1]: Probably don't need this image anymore?

File: src/stylesheets/components/_concept.scss [hunk 2]
```diff
@@ -1,3 +1,23 @@
+@keyframes capeFlow {
+
+  from {
+    d: path("M102.7,114.1c-19.8-8.3-41.8-12.4-61.3-3.2c-16,7.6-29.3,24.2-35.4,44.4c-1.9,6.4-3.2,13-4.4,19.6c-0.9,4.8-1.8,9.8-1.6,14.7c0.2,4.5,1.4,8.8,2.8,12.9c10.3,29.6,33.8,49.2,58.4,57.9s50.6,8,76.1,7.3c15.5-0.4,31.1-0.8,46.1-5.3c31.2-9.2,74.9-39.5,96.2-70.2c2.2-3.2,0.2-7.6-3.5-8H276c-17.4-1.6-47.9-15.2-58.1-33.1c-10.2-17.9-20.2-37.8-36.3-46.4c-6.2-3.3-13.1-4.7-19.8-5c-9.8-0.4-19.7,1.4-29.1,5.4");
+  }
+
+  to {
+    d: path("M105.2,114.1c-19.8-8.3-41.8-12.4-61.3-3.2c-16,7.6-36.9,25.9-43,46.1c-1.9,6.4,1.9,24.5,0.7,31.1c-0.9,4.8-1.8,9.8-1.6,14.7c0.2,4.5,1.4,8.8,2.8,12.9c10.3,29.6,36.2,36,60.8,44.7s50.6,8,76.1,7.3c15.5-0.4,31.1-0.8,46.1-5.3c31.2-9.2,48.8-9.2,70.1-39.9c2.2-3.2,0.2-7.6-3.5-8h-0.1c-17.4-1.6-21.8-45.5-32-63.4c-10.2-17.9-20.2-37.8-36.3-46.4c-6.2-3.3-13.1-4.7-19.8-5c-9.8-0.4-19.7,1.4-29.1,5.4");
```
Comment (jeremyfrank) [inline src/stylesheets/components/_concept.scss:8]: Had no idea you could target the path `d` property in CSS like this!

File: src/stylesheets/components/_concept.scss [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+    fill: #173C4A;
+  }
+}
+
+.cape path {
+  animation: 3s ease-in-out infinite alternate capeFlow;
+}
+
+.cape {
+  transform: translate(20px, 18px);
```
Comment (jeremyfrank) [inline src/stylesheets/components/_concept.scss:18]: Why is the transform necessary here?

**Review by jeremyfrank — APPROVED** [review]
Looks good!

File: src/images/florence.svg (hunk 1 above)
Comment (peytonchance) [reply src/images/florence.svg:1]: The Florence image gets called in the footer as well, so I had to keep it!

File: src/stylesheets/components/_concept.scss (hunk 3 above)
Comment (peytonchance) [reply src/stylesheets/components/_concept.scss:18]: When I pulled in the modified cape paths, they shifted up and left slightly. The transform addresses that issue - maybe it's a little hacky?

**Comment by jeremyfrank** [comment]
Oh, I just checked it out in Firefox and looks like the cape doesn't animate. 🤔

**Review by benjtinsley — APPROVED** [review]
real cool

File: src/stylesheets/components/_concept.scss (hunk 2 above)
Comment (olets) [reply src/stylesheets/components/_concept.scss:8]: cool!

**Pull request merged**